src/types/wrappers/EnzymeConstrainedModel.jl

```diff
@@ -189,15 +189,17 @@ Accessors.reaction_variables(model::EnzymeConstrainedModel) =
 $(TYPEDSIGNATURES)
 
 Get a mapping of enzyme variables to variables -- for enzyme constrained models,
-this is just a direct mapping.
+this is proportional to the molar mass of each gene product. Effectively the
```

Comment:
"to the molar mass of each gene product" is misleading ("molar mass" means "mass per mol" which it is not, and people will ignore the "proportional to" because it's not specified how.)

perhaps say directly

" In EnzymeConstrainedModel(@ref), this represents total mass influx of the enymes" or something.